recipe/meta.yaml

```diff
@@ -40,19 +40,13 @@ test:
     - qpd_ray  # [py<311 and (linux64 or win)]
     - qpd_test
     - qpd_version
-  commands:
-    - pip check
-    # Tests restricted due to modin-ray and ray-defaults unavailability.
-    # Switching off tests for features not implemented yet (NotImplementedError)
-    - pytest -vvv --capture=tee-sys tests -k "not (test_union or test_window_sum_partition_by or test_order_by_limit or test_with)"  # [py<311 and (linux64 or win)]
```

Comment:
whatever happened to the tests?

Reply:
@lorepirri  What if you add `--ignore=tests/unit/qpd_ray`, see https://inspector.pypi.io/project/qpd/0.4.4/packages/81/51/82f1751e3aea61edf0c36e1bb8284e57f0856b550f9bc44be42d61335bf2/qpd-0.4.4.tar.gz/qpd-0.4.4/setup.cfg#line.6

Reply:
@boldorider4 @skupr-anaconda the folder `tests` is not included in the tarball, and there are no sources for this project.

https://inspector.pypi.io/project/qpd/0.4.4/packages/81/51/82f1751e3aea61edf0c36e1bb8284e57f0856b550f9bc44be42d61335bf2/qpd-0.4.4.tar.gz/